### ISSUE TITLE
[FEATURE] Vulcan BoatFly

### ISSUE THREAD

**Issue by EclipsesDev** [CCBlueX/LiquidBounce#4496]
### LiquidBounce Branch

Legacy

### Describe your feature request.

title

### Additional context

no

**Comment by DataM0del** [comment]
just copy the nextgen boatfly and port it to Legacy :)

**Comment by liquidsquid1** [comment]
> just copy the nextgen boatfly and port it to Legacy :)

You’re correct! Eclipses can’t do anything without copying something else, even FDP

**Comment by DataM0del** [comment]
> > just copy the nextgen boatfly and port it to Legacy :)
> 
> You’re correct! Eclipses can’t do anything without copying something else, even FDP

And? What's wrong with copying stuff from nextgen and putting it in legacy.

**Comment by EclipsesDev** [comment]
> > just copy the nextgen boatfly and port it to Legacy :)
> 
> You’re correct! Eclipses can’t do anything without copying something else, even FDP

keep coping bud

"izuna can you skid legacy verusfly disabler?" 
- liquidskid 2024

**Comment by 1zun4** [comment]
https://github.com/CCBlueX/LiquidBounce/pull/5446
